app/lib/page-cache.ts

```diff
@@ -0,0 +1,13 @@
+export const NO_CACHE = {
+  'Cache-Control': 'private, no-store, no-cache, must-revalidate',
+};
+
+export const CACHE_1_DAY = {
+  'Cache-Control': 'public, max-age=0',
+};
+
+export const CACHE_1_HOUR_SWR = {
+  'Cache-Control': 'public, max-age=0, must-revalidate',
+  'Netlify-CDN-Cache-Control':
+    'public, max-age=3600, stale-while-revalidate=60',
+};
```

Comment:
Could also be a good use case for Matt's https://github.com/ascorbic/cdn-cache-control/